### PR TITLE
bitrise.yml normalized + test workflow now runs go test

### PR DESCRIPTION
**Pull request by viktorbenei**
`run-this-step` workflow is the old "test" workflow, which actually runs the step (requires private credentials)

### CODE REVIEW

**Review by bootstraponline — APPROVED** [review]

File: bitrise.yml [hunk 1]
```diff
@@ -98,3 +89,13 @@ workflows:
...
+  test:
+    steps:
+    - script:
+        title: go test
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+
+            go test ./...
```
Comment (bootstraponline) [inline bitrise.yml:101]: would be nice to also run https://github.com/kisielk/errcheck

Comment (viktorbenei) [reply bitrise.yml:101]: definitely! we have a template for Go tools (https://github.com/bitrise-io/bitrise/blob/master/bitrise.yml#L87)

**Comment by viktorbenei** [comment]
But first we should make the test pass ;)

If you have some time, can you change the go test to pass (exit code 0)? You can check the error logs on bitrise.io, just an example from there:

```
Error:      	Received unexpected error:
		
	            	GCLOUD_KEY is not defined!
```

and

```
expected: []string{"gcloud", "firebase", "test", "android", "run", "--type", "instrumentation", "--test", "/tmp/test.apk", "--app", "/tmp/app.apk", "--results-bucket=golang-bucket", "--results-dir=2017-07-18_2:41:33.94349_kgdI", "--device-ids", "NexusLowRes", "--os-version-ids", "25", "--locales", "en", "--orientations", "portrait", "--timeout", "25m", "--directories-to-pull=/sdcard", "--environment-variables", "^:^coverage=true:coverageFile=/sdcard/coverage.ec"}
		
	            	actual: []string{"gcloud", "firebase", "test", "android", "run", "--type", "robo", "--app", "", "--results-bucket=", "--results-dir=2017-07-18_2:41:33.94349_kgdI"}
```

**Comment by viktorbenei** [comment]
Forgot to ping you @bootstraponline in my last comment

**Comment by bootstraponline** [comment]
Fixing the tests now. 😄

**Comment by viktorbenei** [comment]
Awesome, thank you! ;)

**Comment by viktorbenei** [comment]
Merging to make it easier to prepare CI built into this config

**Comment by bootstraponline** [comment]
https://github.com/bitrise-community/steps-firebase-test-lab/pull/10

File: bitrise.yml (hunk 1 above)
Comment (viktorbenei) [reply bitrise.yml:101]: added in https://github.com/bitrise-community/steps-firebase-test-lab/pull/11

**Pull request merged**